lib/activerecord-import/import.rb

```diff
@@ -483,7 +483,8 @@ def import_helper( *args )
 
       # Don't modify incoming arguments
       on_duplicate_key_update = options[:on_duplicate_key_update]
-      if on_duplicate_key_update && on_duplicate_key_update.duplicable?
+      if on_duplicate_key_update && (on_duplicate_key_update.duplicable? || on_duplicate_key_update == :all)
```

Comment:
I don't think we need to change this conditional, because I think it might make more sense to move the code below outside of this if block.

Reply:
I think that this condition is necessary because on_duplicate_key_update argument expects an Array or Hash.

Reply:
```
Book.import(books, on_duplicate_key_update: [:all])
```
In this way I think that it is possible to update all the columns without changing the condition, which do you think is appropriate?

Reply:
I like the way you have it with `on_duplicate_key_update: :all`. That avoids the possible issue of a table having a field named `all`.

Reply:
`if on_duplicate_key_update && on_duplicate_key_update.duplicable?` results in `false` if on_duplicate_key_update is `:all` correct?

If that's the case, I say we just move

`on_duplicate_key_update = attribute_names.reject { |attr| attr == primary_key }.map(&:to_sym) if on_duplicate_key_update == :all` directly below that if block.

Reply:
`if on_duplicate_key_update && on_duplicate_key_update.duplicable?`
This results will be false, because `on_duplicate_key_update.duplicable? ` returns an false if it is not an array or hash
`on_duplicate_key_update` has a `:all`. It is a Symbol.